myst_nbs/case_studies/multilevel_modeling.myst.md

```diff
@@ -1149,7 +1134,3 @@ mcelreath2018statistical
 %load_ext watermark
 %watermark -n -u -v -iv -w
 ```
```

Comment:
The page footer is also missing https://docs.pymc.io/en/latest/contributing/jupyter_style.html#epilogue